docs/Local/installMPS.md

```diff
@@ -99,17 +101,17 @@ Start the MPS, RPS, and Sample Web UI in three separate command line terminals.
 
 **To start the RPS:**
 
-1. Open a new Terminal or Command Prompt and navigate to the `rps` directory inside the `open-amt-cloud-toolkit` directory. 
-
-2. Start the RPS server.
+1. Open a new Terminal or Command Prompt and navigate to the `rps` directory inside the `open-amt-cloud-toolkit` directory. Start the RPS server.
 
     === "Linux"
         ``` bash
+        cd ../rps
```

Comment:
The ..\ is unnecessary since users have to open a new terminal rather than navigate from existing mps directory.